packages/ui/src/GlobalStyle/global.style.ts

```diff
@@ -0,0 +1,17 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;500;600;700&display=swap');
+    * {
+        box-sizing: border-box;
+    }
+    html {
+        font-size: 62.5%;
```

Comment:
lets remove this, we dont really need it

Reply:
I just can see you selected from line 1 to 9. Do you want me to remove line 5 to 10? 

We don't need the `box-sizing: border-box` anymore? I thought that will save us some trouble.

Reply:
I think I selected line 9 only. Didnt i? Now worries, I meant only `font-size: 62.5%;`

Reply:
No problem, I will remove the 62.5% size then. I am also going to go back to using pixels for the button font-size then.